### PR TITLE
Add firefox/privacy/book (#9490, #9520, #9523)

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Adds firefox/privacy/book and updates cards on firefox/more

## Issue / Bugzilla link
#9490
#9520
#9523

## Testing
[Design](https://github.com/mozilla/bedrock/issues/9522#issuecomment-710085508) (Design of this branch has been approved.)
[Copy](https://docs.google.com/document/d/16yJDqxFUqLKAgWTQgjHL6ZE6DaqEwisqLl3ZvQryF_c/edit?usp=sharing)
[Demo](https://www-demo-pr-9596.herokuapp.com/en-US/firefox/privacy/book/)

### CODE REVIEW

**Comment by amychurchwell** [comment]
@rraue in the [design issue](https://github.com/mozilla/bedrock/issues/9522) you mentioned:

> Has 2 cta - download firefox & download pdf in en & de - in all other locales only download as cta

Where are the pfs? We'll have to host them to have the conditional ctas.

**Comment by rraue** [comment]
@amychurchwell The pdf will not be ready when we push the page, so we need to hide these buttons and activate/add them once the pdfs are ready. I just wanted them to be baked in the design so the direction is clear.

File: l10n/en/firefox/privacy/book.ftl [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/privacy/book
+
+# HTML page title
+little-book-of = Little Book of Privacy
```
Comment (alexgibson) [inline l10n/en/firefox/privacy/book.ftl:8]: Please prefix Fluent IDs with a name space.

**Comment by amychurchwell** [comment]
@rraue once the pdfs are ready we can file an issue to add them to this page. 👍

File: l10n/en/firefox/privacy/book.ftl (hunk 1 above)
Comment (amychurchwell) [reply l10n/en/firefox/privacy/book.ftl:8]: Is this an ID? It's only used once in the doc. Perhaps I don't understand.

Comment (alexgibson) [reply l10n/en/firefox/privacy/book.ftl:8]: All Fluent string IDs in this file should be namespaced with a unique prefix (as per the docs and in other files), and just like you have done in #9586

For example: 

```
privacy-book-little-book-of = Little Book of Privacy
privacy-book-how-to-stay = How to stay relaxed and be secure online.
```

Right now there's no namespacing, which means it's hard to work out what file a string belongs to in the template, and also increases the likelihood of creating duplicate IDs over time.

**Comment by amychurchwell** [comment]
Adding "do not merge" until demo approval. Could still benefit from a pair of eyes because it is _quite_ a lot of content.

File: l10n/en/firefox/all.ftl [hunk 2]
```diff
@@ -8,6 +8,7 @@ firefox-all-check-the-system-requirements = Check the system requirements
 firefox-all-release-notes = Release notes
 firefox-all-source-code = Source code
 firefox-all-need-help = Need help?
+firefox-all-learn-more-about-firefox = Learn more about Firefox
```
Comment (rraue) [inline l10n/en/firefox/all.ftl:11]: Same here, this shouldn't be updated, we decided against the link on /all

File: bedrock/firefox/templates/firefox/all-unified.html [hunk 3]
```diff
@@ -90,8 +90,10 @@ <h1 class="c-intro-heading">{{ ftl('firefox-all-choose-which-firefox') }}</h1>
         </div>
         <div class="c-support-links">
           <ul>
+            <li><a href="{{ url('firefox.more') }}">{{ ftl('firefox-all-learn-more-about-firefox') }}</a>
```
Comment (rraue) [inline bedrock/firefox/templates/firefox/all-unified.html:93]: I think this was part of my branch but shouldn't be. We decided against the link there in https://github.com/mozilla/bedrock/issues/9610. It think I just accidentally added this to the /more branch.

File: bedrock/firefox/templates/firefox/more.html [hunk 4]
```diff
@@ -16,18 +16,51 @@
 {% block content %}
 <main>
   {% call hero(
-    title=ftl('firefox-products-are'),
+    title=ftl('learn-more-about-firefox'),
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/more.html:19]: You can keep the old string as a fallback for l10n if you want. Whether or not that makes sense depends on how much work has been done to localize the page.

File: l10n/en/firefox/more.ftl [hunk 5]
```diff
@@ -22,6 +22,14 @@ get-the-privacy = Get the privacy you deserve. Enhanced Tracking Protection make
 take-your-passwords = Take your passwords everywhere
 securely-access-the = Securely access the passwords you’ve saved in { -brand-name-firefox } from anywhere — even outside of the browser.
 choose-which-firefox = Choose which { -brand-name-firefox-browser } to download in your language
+firefox-chromebook = { -brand-name-firefox-browser } for Chromebook
```
Comment (stephaniehobson) [inline l10n/en/firefox/more.ftl:25]: I know it's too late to add the namespace to the existing strings but we could start now with the new ones.

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
This page looks great, especially on XL screens - it switches nicely to mobile too 💯 

Only blocking feedback is an update to the heaving levels. Two other optional things I noted if time/energy allows.

**Review by stephaniehobson — APPROVED** [review]

**Comment by stephaniehobson** [comment]
R+ 🍂 

Thanks for adding the fallback.

**Pull request merged**